.github/workflows/build-main.yaml

```diff
@@ -78,8 +99,9 @@ jobs:
       - name: "Checkout cos-manifests project"
         uses: actions/checkout@v2
         with:
-          repository: bf2fc6cc711aee1a0c2a/cos-manifests
+          repository: hdamarcelo/cos-manifests
```

Comment:
seems a leftover

Reply:
@rinaldodev, yes the change was in the index. I pushed it now.